docs/manual/transform_data.rst

```diff
@@ -0,0 +1,512 @@
+      | dates       integers   floats  strings
+      | date32         int32  float64  str32
+   -- + ----------  --------  -------  -------
+    0 | NA                 4    -13    D
+    1 | 2000-01-05         1     10    A
+    2 | 2010-11-23         2     11.5  B
+    3 | 2020-02-29         3     12.3  NA
+   [4 rows x 4 columns]
+
+Sorting is possible via :ref:`f-expressions`::
+
+   >>>  DT[:, :, dt.sort(f.floats)]
+      | dates       integers   floats  strings
+      | date32         int32  float64  str32
+   -- + ----------  --------  -------  -------
+    0 | NA                 4    -13    D
+    1 | 2000-01-05         1     10    A
+    2 | 2010-11-23         2     11.5  B
+    3 | 2020-02-29         3     12.3  NA
+   [4 rows x 4 columns]
+
+   >>> DT.sort(f.strings)
+      | dates       integers   floats  strings
+      | date32         int32  float64  str32
+   -- + ----------  --------  -------  -------
+    0 | 2020-02-29         3     12.3  NA
+    1 | 2000-01-05         1     10    A
+    2 | 2010-11-23         2     11.5  B
+    3 | NA                 4    -13    D
+   [4 rows x 4 columns]
+
+The default sorting order is ascending; if there are any nulls in the sorting columns, they go to the top.
+
+The sorting order and the position of nulls can be changed in a number of ways:
+
+-  Sorting can be in descending order via the `reverse` parameter::
+
+      >>> DT[:, :, dt.sort('integers', reverse = True)]
+         | dates       integers   floats  strings
+         | date32         int32  float64  str32
+      -- + ----------  --------  -------  -------
+       0 | NA                 4    -13    D
+       1 | 2020-02-29         3     12.3  NA
+       2 | 2010-11-23         2     11.5  B
+       3 | 2000-01-05         1     10    A
+      [4 rows x 4 columns]
+
+.. note::
+
+   The ``reverse`` parameter is available only in the :func:`sort()` function
+
+- Sorting in descending order is possible by negating the :ref:`f-expressions` within the :func:`sort()` function, or the :meth:`datatable.Frame.sort` method::
+
+      >>> DT[:, :, dt.sort(-f.integers)]
+         | dates       integers   floats  strings
+         | date32         int32  float64  str32
+      -- + ----------  --------  -------  -------
+       0 | NA                 4    -13    D
+       1 | 2020-02-29         3     12.3  NA
+       2 | 2010-11-23         2     11.5  B
+       3 | 2000-01-05         1     10    A
+      [4 rows x 4 columns]
+
+
+      >>> DT.sort(-f.integers)
+         | dates       integers   floats  strings
+         | date32         int32  float64  str32
+      -- + ----------  --------  -------  -------
+       0 | NA                 4    -13    D
+       1 | 2020-02-29         3     12.3  NA
+       2 | 2010-11-23         2     11.5  B
+       3 | 2000-01-05         1     10    A
+      [4 rows x 4 columns]
+
+- The position of null values within the sorting column can be controlled with the ``na_position`` parameter::
```

Comment:
If you want, you can also mention `na_position="remove"` that removes all the rows that contain missings.

Reply:
totally missed the `remove` option. WIll add an example for that